fmriprep/interfaces/bids.py

```diff
@@ -423,8 +423,11 @@ def _copy_any(src, dst):
     dst_isgz = dst.endswith('.gz')
     if src_isgz == dst_isgz:
         copyfile(src, dst, copy=True, use_hardlink=True)
+        return True
+
     src_open = gzip.open if src_isgz else open
     dst_open = gzip.open if dst_isgz else open
     with src_open(src, 'rb') as f_in:
         with dst_open(dst, 'wb') as f_out:
             copyfileobj(f_in, f_out)
+    return True
```

Comment:
Unless there's a use for `True` (and some case in which we might return `False`), I would just use `return` with no argument.

Reply:
We could add one output for debugging: return `False` if no compress/uncompress was performed, `True` otherwise. WDYT?

Reply:
Sure. Maybe we should go ahead and add a unit test for this.

Reply:
(I recognize that I'm using "we" somewhat disingenuously. I can't stop.)

Reply:
Let's make it a real "we". @mgxd would you like to add that unit test?

(actually, we probably may want to see this interface in nipype proper, WDYT? cc/ @satra)